### ISSUE TITLE
ES module error after upgrading to 0.4.1

### ISSUE THREAD

**Issue by curthipster** [hosseinmd/prettier-plugin-jsdoc#174]
After upgrading to 0.4.1, prettier is failing with the following error:

``` bash
% ./node_modules/.bin/prettier . --check
[error] require() of ES Module [path redacted]/node_modules/mdast-util-from-markdown/index.js from [path redacted]/node_modules/prettier-plugin-jsdoc/dist/index.js not supported.
[error] Instead change the require of [path redacted]/node_modules/mdast-util-from-markdown/index.js in [path redacted]/node_modules/prettier-plugin-jsdoc/dist/index.js to a dynamic import() which is available in all CommonJS modules.
```

**Comment by cailloumajor** [comment]
Version 0.4.1 looks like a phantom release: there is no mention of it in the repository (commit history, changelog, `version` property in `package.json`, ...). I would argue (at least for myself) to not update to it.

**Comment by hosseinmd** [comment]
Fixed: v0.4.2
